setup.cfg

```diff
@@ -50,6 +50,9 @@ testpaths = tests
 xfail_strict = true
 markers =
     array_compare: for functions using array comparison
+filterwarnings =
+    error
+    ignore:distutils Version classes are deprecated
```

Comment:
do add a comment for ignore lines like this, e.g. which upstream version should be the minimal when this could be removed.